Inputs/SplineValidations.yaml

```diff
@@ -0,0 +1,42 @@
+Systematics:
+ - Systematic:
+     Names:
+       FancyName: Spline_Param_0
+     Error: 0.06
+     FlatPrior: false
+     IsFixed: false
+     Mode: []
+     ParameterBounds: [0.0, 9999]
+     ParameterGroup: Xsec
+     ParameterValues:
+       Generated: 1.21
+       PreFitValue: 1.03
+     SplineInformation:
+       Mode: [0]
+       InterpolationType: TSpline3
+       SplineName: Spline_0
+     DetID: 985
+     StepScale:
+       MCMC: 0.4
+     Type: Spline
+
+ - Systematic:
+     Names:
+       FancyName: Spline_Param_1
+     Error: 1.0
+     FlatPrior: True
+     IsFixed: false
+     Mode: []
+     ParameterBounds: [0., 1.]
+     ParameterGroup: Xsec
+     ParameterValues:
+       Generated: 1.0
+       PreFitValue: 1.0
+     SplineInformation:
+       Mode: [9]
```

Comment:
It might be me being pedantic but this isb\t used right now.

I would remove it or move to CIValdiations fodler where I don't care so much about tidy

Reply:
yep sure I'll move this then. I actually made this because I though the SplineValidations script read in a yaml or used a covarianceXSec but it actually just has the spline names hard-coded.